flask_restful/reqparse.py

```diff
@@ -236,10 +243,17 @@ def add_argument(self, *args, **kwargs):
         See :class:`Argument`'s constructor for documentation on the
         available options.
         """
+
         if len(args) == 1 and isinstance(args[0], self.argument_class):
             self.args.append(args[0])
         else:
             self.args.append(self.argument_class(*args, **kwargs))
+
+        #Do not know what other argument classes are out there
+        if self.trim and type(self.argument_class) is type(Argument):
```

Comment:
Oh yea, however isinstance needs an instance object for comparison, therefore I would have to `isinstance(Argument(name='arg'), self.argument_class)`. I ended up using correct comparison: `if self.argument_class is Argument`